CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ _May 27, 2020_
   Please make sure to provide a valid operator.
   Example:  `.custom("contains", key: "teams", value: "red")`
 - `queryUsers` now supports `Pagination.limit` and `Pagination.offset` [#288](https://github.com/GetStream/stream-chat-swift/issues/288)
+- `disableLocalNotifications` added to `Notifications` for disabling local notifications [#290](https://github.com/GetStream/stream-chat-swift/pull/290)
```

Comment:
We had a release, please move this changes in the Upcoming section.

Reply:
Oops. :) done